wiki/Guides/BSS_Issues/fr.md

```diff
@@ -0,0 +1,43 @@
+---
+tags:
+- I can't submit my own beatmap!
+- beatmap submission system issues
+---
+
+# Problème avec le SSB
+
+*Note: N’utilisez pas ce guide pour voler les beatmaps d’autres créateurs !*
+
+Si le [SSB](/wiki/BSS) (Système de Soumission de Beatmap) ne vous laisse pas soumettre votre propre beatmap car il dit que ce n’est pas votre propre beatmap puis faites ce qui suit :
+
+1.  Aller dans le dossier beatmapset.
+    -    Vous pouvez y accéder facilement en allant dans `File` puis `Open Song Folder` dans l’[éditeur de beatmap](/wiki/beatmap_editor).
+2.  Sortez d’osu! (Pour être sur que ces changements seront appliqués proprement.)
+3.  Ouvrez le fichier difficulté `.osu` avec un éditeur de texte brut (ex : Notepad).
+4. Entrez votre nom d’utilisateur pour le  `Creator`. (Si votre nom a des espaces, utilisez des espaces.)
+5.  Mettre le  `BeatmapID` à `0`.
+6.  Mettre le `BeatmapSetID` à `-1`.
+7.  Soyez sûr que le nom de dossier de la beatmap ne commence pas avec une une série de nombres.
+    -    Ex : `1000 - Song Name` devrait être `Song Name`.
+8.  Supprimer le/les fichier(s) dans le dossier `SubmissionCache`. Ceci est localisé dans le dossier d’installation d’osu!.
+9.  Ouvrez à nouveau osu! et tentez de télécharger votre beatmap.
+10.  Si cela échoue, essayez les étapes ci-dessous.
+
+---
+
+Si vous avez toujours des problèmes, suivez ces étapes :
```

Comment:
```suggestion
Si vous avez toujours des problèmes, suivez ces étapes :
```
Je sais pas si cette phrase est utile a remettre étant donné que tu l'annonce juste au dessus par "Si cela échoue, essayez les étapes ci-dessous."

Reply:
C'est écrit dans l'article original ^^'

Reply:
Je vais voir avec clayton pour eventuellement changer ça dans l'article original, car pour moi ça a pas de sens de le mettre deux fois, surtout a la suite.

Reply:
Yep je comprend

Reply:
I removed the one in the list instead